src/main/res/values/styles.xml

```diff
@@ -7,7 +7,7 @@
         <item name="android:windowIsFloating">false</item>
         <item name="android:windowContentOverlay">@null</item>
         <item name="android:windowSoftInputMode">stateAlwaysHidden</item>
-        <item name="android:windowActionModeOverlay">true</item>
+        <item name="windowActionModeOverlay">true</item>
```

Comment:
What was wrong with this?

Reply:
That's what is changed when migrating app to appCompat

Reply:
Okay, apparently needed for compat activity. See if you have any deviations in how activity looks like in plugins.